src/main/java/com/conveyal/taui/controllers/RegionalAnalysisController.java

```diff
@@ -25,9 +26,7 @@
 import java.io.IOException;
 import java.util.Collection;
 
-import static spark.Spark.delete;
```

Comment:
Is this wildcard import because of an IntelliJ default (Editor -> Code Style -> Java)?